client/src/components/CreateAQuestion/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { createRef, useRef } from 'react';
+// import { useGameContext } from '../../utils/GlobalState';
+import { ADD_QUESTION } from '../../utils/actions';
+import Button from "../Button";
+import Toggle from "../Toggle";
+
+const CreateAQuestion = () => {
+    const topicRef = useRef();
```

Comment:
@dropcat13 to ensure our future selves love us, we may want to use the same values here that will eventually find their way into the db: `categoryRef`, `difficultyRef`